lint/pom.xml

```diff
@@ -0,0 +1,98 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.datacommons</groupId>
+        <artifactId>datacommons-import</artifactId>
+        <version>1.0-SNAPSHOT</version>
+        <relativePath>..</relativePath>
+    </parent>
+
+    <groupId>org.datacommons</groupId>
+    <artifactId>datacommons-import-lint</artifactId>
+    <name>Data Commons - Import Lint Tool</name>
+    <url>http://www.datacommons.org</url>
```

Comment:
very nit: could just be https://datacommons.org (www will be redirected)

Reply:
Done.